cli/azd/pkg/project/framework_service.go

```diff
@@ -26,17 +27,14 @@ const (
 )
 
 func parseServiceLanguage(kind ServiceLanguageKind) (ServiceLanguageKind, error) {
-	if string(kind) == "" {
-		return ServiceLanguageKind(""), fmt.Errorf("language property must not be empty")
-	}
-
 	// aliases
 	if string(kind) == "py" {
 		return ServiceLanguagePython, nil
 	}
 
 	switch kind {
-	case ServiceLanguageDotNet,
+	case ServiceLanguageNone,
```

Comment:
I think we're not yet exposing `language == none` as an extended feature, although it'd be interesting how it plays with #1645. Just as a polish, can we make sure we provide a proper parse failure error `if language == "" && image == ""` in `project.Parse`?